src/app/clients/ActionBar.cy.tsx

```diff
@@ -0,0 +1,84 @@
+import { Schema } from "@/supabase";
+import ActionBar from "@/app/clients/ActionBar";
+import React from "react";
+import StyleManager from "../themes";
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import "dayjs/locale/en-gb";
+
+describe("Clients - Action Bar", () => {
+    const mockData: Schema["parcels"][] = [
+        {
+            primary_key: "1",
+            client_id: "1",
+            collection_centre: "Clapham",
+            collection_datetime: null,
+            voucher_number: null,
+            packing_datetime: null,
+        },
+    ];
+
+    const selectedIndices = [0];
+
+    const Component: React.FC<{}> = () => {
+        return (
+            <LocalizationProvider adapterLocale="en-gb" dateAdapter={AdapterDayjs}>
+                <StyleManager>
+                    <ActionBar data={mockData} selected={selectedIndices} />
+                </StyleManager>
+            </LocalizationProvider>
```

Comment:
Shouldn't the localization provider already be added in themes? or is it different because its a cypress file?